### ISSUE TITLE
spread inside generator accesses undeclared variable

### ISSUE THREAD

**Issue by monsanto** [babel/babel#1268]
```
function *x() {
    [].concat(...[])
}
```

-->

```
function x() {
    return regeneratorRuntime.wrap(function x$(context$1$0) {
        while (1) switch (context$1$0.prev = context$1$0.next) {
            case 0:
                (_ref = []).concat.apply(_ref, []);

            case 1:
            case "end":
                return context$1$0.stop();
        }
    }, marked0$0[0], this);
}
```

note the undeclared `_ref`.

Previously: https://github.com/babel/babel/issues/438, https://github.com/babel/babel/issues/803

**Comment by sebmck** [comment]
Gah, it's because of the change to vanilla regenerator.

**Comment by sebmck** [comment]
Sorry for the delay, fixed as of 5.2.0. Thanks!
